spec/cucumber/cli/rerun_spec.rb

```diff
@@ -71,16 +71,17 @@ module Cli
         end
       end
 
-      context "contains features and scenarios with and without spaces same line and across lines" do
+      context "contains features and scenarios with and without spaces on same line and across lines" do
```

Comment:
@mattwynne - yep fair point, will break them out a bit - probably this should also be fixed in parallel_tests but since we don't constrain users via a known interchange format (json/yaml whatever) I guess there is no harm in defending against 🎃 👻 formats..  I'll break up the specs a bit.